src/main/pg/rx.h

```diff
@@ -24,6 +24,9 @@
 
 #include "pg/pg.h"
 
+#define GET_RSSI_LPF_FREQUENCY(period) (1 / (period / 10.0f))
+#define RSSI_RESAMPLE_US 100000
```

Comment:
Defines should be renamed to reflect that they're related to frame errors and not true RSSI.

Reply:
Done